gestalt/vault.py

```diff
@@ -55,12 +67,19 @@ def __init__(self,
             try:
                 hvac.api.auth_methods.Kubernetes(
                     self.vault_client.adapter).login(role=role, jwt=jwt)
-                token = self.vault_client.auth.token.lookup_self()
+                token = retry_call(
+                    self.vault_client.auth.token.lookup_self,
+                    exceptions=(RuntimeError, Timeout),
+                    delay=self.delay,
+                    tries=self.tries,
+                )
+
                 if token is not None:
                     kubes_token = (
                         "kubernetes",
-                        token['data']['id'],  # type: ignore
-                        token['data']['ttl'])  # type: ignore
+                        token["data"]["id"],  # type: ignore
```

Comment:
Is there a reason why we switched from single quotes to double quotes? This is your local editor/linter request.